distributed/__init__.py

```diff
@@ -1,42 +1,42 @@
-from . import config
+from . import config  # isort:skip
```

Comment:
Could you add a small comment here about why we need to skip `isort`?

Reply:
Sure!